torch/onnx/_internal/exporter/_onnx_program.py

```diff
@@ -30,6 +30,15 @@
     import onnxruntime as ort
 
 _LARGE_MODEL_THRESHOLD = 1536 * 1024 * 1024  # 1536MB
+_NP_UNSUPPORTED_DTYPES = frozenset(
```

Comment:
We should better create an independent python file to collect these scattered dtype tables.

Reply:
They are all slightly different and not reused so I am creating them locally. If we see enough reuse I can move them then